src/controllers/contact/contact.controller.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  Controller,
+  Post,
+  Req,
+  Body,
+  HttpException,
+  HttpStatus,
+} from '@nestjs/common';
+import {
+  ApiOperation,
+  ApiResponse,
+  ApiTags,
+  ApiBearerAuth,
+} from '@nestjs/swagger';
+import { ContactPresenter } from './contact.presenter';
+import { Request } from 'express';
+import { CreateContactDto } from './contact.dto';
+
+// Slack の Incoming Webhook URL
+const webhookUrl = process.env.SLACK_WEBHOOK_URL;
+
+@ApiTags('contact')
+@ApiBearerAuth()
+@Controller('contact')
+export class ContactController {
+  @Post()
+  @ApiOperation({ operationId: 'contact' })
+  @ApiResponse({
+    status: 200,
+    description: '問い合わせ内容を送信する',
+    type: ContactPresenter,
+  })
+  async postContact(
```

Comment:
API的にはメッセージを送信するから、sendのほうがいいかも
```suggestion
  async sendContact(
```